app/views/orgs/admin_edit.html.erb

```diff
@@ -6,71 +6,80 @@
+      <div class="col-sm-8">
+        <%= f.text_field :abbreviation, id: "org_abbreviation", class: "form-control", "aria-required": true %>
+      </div>
+    </div>
+
+    <% if @org.logo.present? %>
+      <div  class="form-group">
+        <%= f.label :logo, 'Organization Logo', class: "col-sm-2 control-label" %>
+        <%= image_tag @org.logo.url, alt: "#{@org.name} #{_('logo')}" %>
+        <div class="form-group">
+          <%= f.label _('Remove logo') % {application_name: Rails.configuration.branding[:application][:name]}, for: :remove_logo, class: "col-sm-2 control-label" %>
+          <div class="checkbox">
+            <%= f.check_box :remove_logo, title: _("This will remove your organisation's logo") %>
           </div>
-        <% end %>
-      
-        <div class="form-input" id="banner-text">
-          <%= f.label _('Top banner text'), for: @org.banner_text %>
-          <div class="inline">
-            <%= text_area_tag('org_banner_text', @org.banner_text, id: "org_banner_text") %>
-            <script>
-              /* Initialises an editor for textarea defined above with id org_banner_text */
-              dmproadmap.utils.tinymce.init({ selector: '#org_banner_text' });
-            </script>
+          <br />
+          <div>
+            <strong> - <%= _('or') %> - </strong>
           </div>
+          <br />
+          <%= f.file_field :logo %>
         </div>
-      
-        <div class="form-input">
-          <%= f.label :target_url %>
-          <%= f.text_field :target_url, class: "left-indent required input-large top-indent" %>
-        </div>
-        <div class="form-input">
-          <%= f.label :contact_email %>
-          <%= f.text_field :contact_email, class: "left-indent required input-medium" %>
+      </div>
+    <% else %>
+      <div class="form-group">
+        <%= f.label :logo, 'Organization Logo', class: "col-sm-2 control-label"  %>
+        <div class="col-sm-8">
+          <%= f.file_field :logo %>
         </div>
-      
-        <div class="form-input">
-          <div class="button-spacer">&nbsp;</div>
-          <%= render partial: 'shared/accessible_submit_button',
-                     locals: {id: 'save_org_submit', 
-                              val: 'Save', 
-                              disabled_initially: true,
-                              tooltip: _('Name and abbreviation are required'),
-                              classes: 'left-indent'} %>
-        </div>
-      </fieldset>
+      </div>
     <% end %>
+
+    <div class="form-group" id="banner-text">
+      <%= f.label _('Top banner text'), for: @org.banner_text, class: "col-sm-2 control-label" %>
+      <div class="col-sm-8">
+        <%= text_area_tag('org_banner_text', @org.banner_text, id: "org_banner_text", class: "form-control") %>
+        <script>
+          /* Initialises an editor for textarea defined above with id org_banner_text */
+          dmproadmap.utils.tinymce.init({ selector: '#org_banner_text' });
```

Comment:
We don't want JS embedded in the HTML anymore. Move this to the page's corresponding JS file. See @jollopre example of wiring up a tinymce editor on the views/answers/ JS files